From 86a7b6ed2700a7db7eb488a5df443d4ae01c7683 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lukas=20Gillsj=C3=B6?= <lukasg@axis.com>
Date: Tue, 12 Oct 2021 23:55:51 +0200
Subject: [PATCH] Set docker proxy via build-arg

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c7cd97f79a..7bba0edd3f 100644
--- a/Makefile
+++ b/Makefile
@@ -146,7 +146,7 @@ ifdef USE_BUILDX
 BUILD_OPTS += $(BUILDX_BUILD_EXTRA_OPTS)
 BUILD_CMD := $(BUILDX) build
 else
-BUILD_CMD := $(DOCKER) build
+BUILD_CMD := $(DOCKER) build --build-arg HTTP_PROXY=${HTTP_PROXY} --build-arg HTTPS_PROXY=${HTTPS_PROXY}
 endif
 
 # This is used for the legacy "build" target and anything still depending on it
-- 
2.25.1

